src/utils/tests.tsx

```diff
@@ -8,13 +8,49 @@ import { Instance } from "../data/entities/Instance";
 import { UserProps } from "../domain/entities/UserProps";
 
 export function getTestUser(): UserProps {
-    // @ts-ignore TODO
     return {
         id: "xE7jOejl9FI",
         name: "John Traore",
         username: "admin",
         userGroups: [],
         userRoles: [],
+        organisationUnits: [],
+        dataViewOrganisationUnits: [],
+        searchOrganisationsUnits: [],
```

Comment:
`searchOrganisationsUnits` looks off, two plurals, is that a real attr? I think latest DHIS2 use `teiSearchOrganisationUnits`, can you confirm?